docs/eventing/samples/writing-event-source-easy-way/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:10
+
+# Create app directory
+WORKDIR /usr/src/app
+
+# Install app dependencies
+# A wildcard is used to ensure both package.json AND package-lock.json are copied
+# where available (npm@5+)
```

Comment:
If we're using node:10, this would be later than npm@5, right?

Reply:
Deleted in https://github.com/knative/docs/pull/2494